pkg/config/config.go

```diff
@@ -117,6 +118,10 @@ type NsxConfig struct {
 	EnvoyHost                 string   `ini:"envoy_host"`
 	EnvoyPort                 int      `ini:"envoy_port"`
 	LicenseValidationInterval int      `ini:"license_validation_interval"`
+	UseAVILB                  bool     `ini:"use_avi_lb"`
+	UseNativeLoadBalancer     *bool    `ini:"use_native_loadbalancer"`
+	RelaxScaleValidaion       bool     `ini:"relax_scale_validation"`
```

Comment:
Name it `RelaxNSXLBScaleValication`. And `NSXLBSize`

Reply:
Fixed.